pkgs/applications/graphics/loupe/default.nix

```diff
@@ -0,0 +1,61 @@
+{ stdenv
+, lib
+, fetchgit
+, rustPlatform
+, meson
+, pkg-config
+, cmake
+, gtk4
+, itstool
+, desktop-file-utils
+, ninja
+, gnome-builder
+, wrapGAppsHook4
+, libadwaita
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "loupe";
+  version = "44.alpha";
```

Comment:
According to https://nixos.org/nixpkgs/manual/#sec-package-naming, the `version` should be `unstable-yyyy-mm-dd`, when using untagged git snapshots.

Reply:
I usually go with that but loupe has it's own version listed so I went with that but I can change it